README.md

```diff
@@ -93,6 +93,7 @@ Gravis GamePad Pro           | 10      | 2     | 0    | 0001  | GrIP       |
 Logitech WingMan Extreme     | 6       | 3     | 1    | 1001  | ADI        |
 Logitech CyberMan 2          | 8       | 6     | 0    | 1001  | ADI        |
 Logitech WingMan Interceptor | 9       | 3     | 3    | 1001  | ADI        | 2 hats are mapped as 4 axes
+Logitech ThunderPad Digital  | 12      | 0     | 0    | 1001  | ADI        | Directional buttons mapped as 2 axes
```

Comment:
```suggestion
Logitech ThunderPad Digital  | 8       | 2     | 0    | 1001  | ADI        | Directional buttons mapped as 2 axes
```

Reply:
In this table we have the devices in the way they are report through USB, not the actual physical configuration. Due to the workaround in the code we map some buttons to the axes, which will be reported as 8 buttons + 2 axes.